app/templates/owner/addWorkSchedule.html

```diff
@@ -0,0 +1,42 @@
+{% extends "owner/layout.html" %}
+{% block title %}Add work schedule{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style>
+    </style>
+{% endblock %}
+{% block body %}
+    <h1>Add work schedule</h1>
```

Comment:
Nazwa plik html to addWorkSchedule.html  nie warto by było tego nazwać po prosty add skoro i tak tylko dodajecie Schedul albo add_schedul (konwencja pythonowa). Nigdzie na świecie nikt nie nazywa tak plików addWorkSchedule, to nie jest metoda w java.